library/core/src/char/methods.rs

```diff
@@ -1250,7 +1250,7 @@ impl char {
     /// let percent = '%';
     /// let space = ' ';
     /// let lf = '\n';
-    /// let esc: char = 0x1b_u8.into();
+    /// let esc: char = '\x1b';
```

Comment:
```suggestion
    /// let esc = '\x1b';
```
You could now also remove all of the `char` type annotations.